iceoryx_examples/callbacks_in_c/ice_c_callbacks_publisher.c

```diff
@@ -51,7 +51,8 @@ void sending()
     {
         if (counter % 3 == 0)
         {
-            if (iox_pub_loan_chunk(publisherLeft, (void**)&chunk, sizeof(struct CounterTopic))
+            if (iox_pub_loan_chunk(
+                    publisherLeft, (void**)&chunk, sizeof(struct CounterTopic), IOX_C_DEFAULT_USER_PAYLOAD_ALIGNMENT)
```

Comment:
Great, the new argument will break my work were I use it I think...

Reply:
of course it will. I'm open for suggestions. Shall I create a second function like `iox_pub_loan_aligned_chunk`?

Reply:
I do not think so, calling this without alignment will lead to subtle errors, better to force the user to do so.

Reply:
There wouldn't be much code duplication, since `iox_pub_loan_chunk` would call `iox_pub_loan_aligned_chunk`. The more difficult question would be if we call it `iox_pub_loan_aligned_chunk` or `iox_pub_loan_chunk_with_alignment`?

Reply:
@MatthiasKillat  elfenpiff had a third option. He suggested to use a variadic function. What do you think about that. His second choice would be `iox_pub_loan_aligned_chunk`

Reply:
forget the variadic function. We can not determine if the optional parameter is set. Therefore it is `iox_pub_loan_aligned_chunk`